src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  height: 16px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:nth-child(5) {
+  margin-right: 0;
+}
+
+.stars__star--active {
```

Comment:
Use pseudoclasses to change the background image for active stars according to the modifier of the block, so then if I change the modifier of the block - the amount of the active stars changes too